park-dashboard/src/Components/Searchbar/Searchbar.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from "react";
+import Select from "react-select";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import "./searchbar.css"
+
+const BASE_URL = "https://developer.nps.gov/api/v1";
+const API_KEY = "OaR7jmqSa22JAcsym9lVfStp58LmCqH9JdZUPEH7";
+
+const Searchbar = () => {
+  const [parkName, setParkName] = useState([]);
```

Comment:
Since it is an array, it would be better if you name it as parkNames.

Reply:
Thanks for suggesting this, the code is now changed.